[FEATURE] Auto tap uband recommendation (#619)
PowerGridModel/power-grid-model: docs/user_manual/components.md

The following additional requirements exist on the input parameters.

- The automatic tap changer algorithm currently only supports tap changers connected at HV side of the transformer. Hence, the rated voltage of the node at the `tap_side` must be greater than or equal to the rated voltage of the node at the other side of transformer.

> - Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$

Author: ```suggestion
- Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: At zero line drop compensation if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```

> - The voltage band must be sufficiently large. Although it depends on the resultant voltage being transformed, if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$

Reviewer: ```suggestion
- Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```

> - The voltage band must be sufficiently large. It would depend on the resultant voltage being transformed. But generally when the resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$

Reviewer 2: ```suggestion
- The voltage band must be sufficiently large. Although it depends on the resultant voltage being transformed, if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```
